metagraph/src/common/vectors/vector_algorithm.hpp

```diff
@@ -15,26 +16,14 @@ sdsl::bit_vector to_sdsl(const std::vector<bool> &vector);
 sdsl::bit_vector to_sdsl(const std::vector<uint8_t> &vector);
 
 template <class Vector>
-sdsl::int_vector<> pack_vector(const Vector &vector, uint8_t bits_per_number) {
-    if constexpr(std::is_same_v<Vector, sdsl::int_vector<>>) {
-        if (bits_per_number == vector.width())
-            return vector;
-    }
-
-    sdsl::int_vector<> packed(vector.size(), 0, bits_per_number);
-    for (uint64_t i = 0; i < vector.size(); ++i) {
-        packed[i] = vector[i];
-    }
-    return packed;
-}
+inline sdsl::int_vector<> pack_vector(const Vector &vector, uint8_t bits_per_number);
```

Comment:
don't overdo :) this was a small function, the implementation helps to see what is the complexity. Here it's better to keep it here.